pkg/image/common/mutator.go

```diff
@@ -109,3 +114,50 @@ func (m *vmiMutator) PatchImageSCParams(vmi *harvesterv1.VirtualMachineImage) ([
 	patchOps = append(patchOps, fmt.Sprintf(`{"op": "%s", "path": "/spec/storageClassParameters", "value": %s}`, verb, string(valueBytes)))
 	return patchOps, nil
 }
+
+func (m *vmiMutator) PatchDefaultSC(vmi *harvesterv1.VirtualMachineImage) ([]string, error) {
```

Comment:
Nit: I think this method is more like `EnsureTargetSC()`, per the inline comments at 145 and 156, because we want to make sure the target sc class is added to the annotation and the spec.

Reply:
yeah, the `EnsureTargetSC()` is more suitable. updated.